rcl/include/rcl/context.h

```diff
@@ -146,7 +148,10 @@ typedef struct rcl_context_t
    * See this paper for an effort to make this possible in the future:
    *   http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p0943r1.html
    */
-  RCL_ALIGNAS(8) uint8_t instance_id_storage[RCL_CONTEXT_ATOMIC_INSTANCE_ID_STORAGE_SIZE];
+  /// @cond Doxygen_Suppress
+  RCL_ALIGNAS(8)
+  /// @endcond
```

Comment:
Is it possible to use a Doxyfile `PREDEFINE` for this, rather than using `cond`?  If not, no big deal, it would just be nicer.

Reply:
Done in https://github.com/ros2/rcl/pull/932/commits/beda62e00bd1e8c320ff67171146c11df61b2c27.